float64[6] pose
---
bool success
string message
